sites/partners/src/components/applications/PaperApplicationForm/sections/FormHouseholdMembers.tsx

```diff
@@ -170,7 +170,7 @@ const FormHouseholdMembers = ({
       </Drawer>
 
       <Dialog
-        isOpen={!!membersDeleteModal}
```

Comment:
**note:** When the modal number was set to zero (when app submitted on public side) this would be false